tikv-configuration-file.md

```diff
@@ -894,6 +894,13 @@ Configuration items related to `rocksdb.defaultcf`
 + Optional values: ["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
 + Default value: `No` for the first two levels, and `lz4` for the next five levels
 
+### `bottommost-level-compression`
+
++ Sets the compression algorithm of the bottom most layer. This configuration item overrides the setting in the `compression-per-level`.
++ The bottom most layer does not directly adopt the last compression algorithm in the `compression-per-level` array from when the data is written to LSM tree; therefore, `bottommost-level-compression` enables the bottom most layer to use the compression algorithm with the best compression effect from the beginning.
```

Comment:
```suggestion
+ Ever since data is written to LSM-tree, TiKV does not directly adopt the last compression algorithm specified in the `compression-per-level` array for the bottommost layer. `bottommost-level-compression` enables the bottommost layer to use the compression algorithm of the best compression effect from the beginning.
```